imap_processing/utils.py

```diff
@@ -101,17 +101,22 @@ def convert_raw_to_eu(dataset: xr.Dataset, conversion_table_path, packet_name):
     return dataset
 
 
-def create_dataset(packets: list[Packet], met_name="shcoarse") -> xr.Dataset:
+def create_dataset(
+    packets: list[Packet], met_name="shcoarse", include_header=True, skip_keys=None
+) -> xr.Dataset:
     """Create dataset for each metadata field.
 
-    # TODO: change IMAP-Hi to use this and others as needed.
-
     Parameters
     ----------
     packets : list[Packet]
         packet list
-    met_name : str
-        metadata name to use as epoch time
+    met_name : str, Optional
+        Default is "shcoarse" because many instrument uses it.
+        This key is used to get spacecraft time for epoch dimension.
+    include_header: bool, Optional
+        Whether to include CCSDS header data in the dataset
+    skip_keys: list, Optional
+        Keys to skip in the metadata
```

Comment:
What is `metadata` in this context? I think it means that matching fields in the input packets are not included in the output dataset?

Reply:
Here, I am using metadata as keys from packet definition after CCSDS header keys. I appreciate other term suggestion.